apps/nfdiv/nfdiv-cron-regen-co-pronounced-cover-letter/nfdiv-cron-regen-co-pronounced-cover-letter.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: nfdiv-cron-regen-co-pronounced-cover-letter
+spec:
+  releaseName: nfdiv-cron-regen-co-pronounced-cover-letter
+  values:
+    job:
+      image: hmctspublic.azurecr.io/nfdiv/case-api:prod-1972fb3-20240722115548 #{"$imagepolicy": "flux-system:nfdiv-case-api"}
+      keyVaults:
+        nfdiv:
+          secrets:
+            - name: app-insights-connection-string
+              alias: app-insights-connection-string
+            - name: uk-gov-notify-api-key
+              alias: UK_GOV_NOTIFY_API_KEY
+            - name: s2s-case-api-secret
+              alias: S2S_SECRET
+            - name: idam-secret
+              alias: IDAM_CLIENT_SECRET
+            - name: idam-systemupdate-username
+              alias: IDAM_SYSTEM_UPDATE_USERNAME
+            - name: idam-systemupdate-password
+              alias: IDAM_SYSTEM_UPDATE_PASSWORD
+      environment:
+        TASK_NAME: SystemRegenerateCoPronouncedCoverLetterOfflineConfidentialTask
```

Comment:
wondering if this will baulk intil that task exists in master?
Could try an existing task name here just to test the config